doc/how-to/getting_started_tcm.rst

```diff
@@ -0,0 +1,359 @@
+
+Deploying the cluster locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To deploy a local cluster based on the configuration from etcd:
+
+#.  Open a system terminal.
+#.  Create a new ``tt`` environment in a directory of your choice:
+
+    .. code-block:: console
+
+        $ mkdir cluster-env
+        $ cd cluster-env/
+        $ tt init
+
+#.  Inside the ``instances.enabled`` directory of the created tt environment, create the ``cluster`` directory.
+
+    .. code-block:: console
+
+        $ mkdir instances.enabled/cluster
+        $ cd instances.enabled/cluster/
+
+#.  Inside ``instances.enabled/cluster``, create the ``instances.yml`` and ``config.yaml`` files:
+
+    *   ``instances.yml`` specifies instances to run in the current environment. In this example, there are three instances:
+
+        ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/tcm_get_started_tt/instances.yml
+            :language: yaml
+            :dedent:
+
+    *   ``config.yaml`` instructs ``tt`` to load the cluster configuration from etcd.
+        The specified etcd location matches the configuration storage of the **Default cluster** in TCM:
+
+        ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/tcm_get_started_tt/config.yaml
+            :language: yaml
+            :dedent:
+
+#.  Start the cluster from the ``tt`` environment root (the ``cluster-env`` directory):
+
+    .. code-block:: console
+
+        $ tt start
+
+To check how the cluster started, run ``tt status``. This output should look like this:
+
+    .. code-block:: console
+
+        $ tt status
+        INSTANCE               STATUS      PID
+        myapp:instance-001     RUNNING     2058
+        myapp:instance-002     RUNNING     2059
+        myapp:instance-003     RUNNING     2060
+
+Managing the cluster in TCM
+---------------------------
+
+To learn to interact with a cluster in |tcm|, complete typical database tasks such as:
+
+*   Checking the cluster state.
+*   Creating a space.
+*   Writing data.
+*   Viewing data.
+
+Checking state
+~~~~~~~~~~~~~~
+
+To check the cluster state in |tcm|, go to **Stateboard**. Here you see the overview
+of the cluster topology, health, memory consumption, and other information.
+
+.. image:: images/tcm_start_stateboard_online.png
+    :width: 700
+    :align: center
+    :alt: Online cluster stateboard
+
+Connecting to the instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To view detailed information about an instance, click its name in the instances list
+on the **Stateboard** page.
+
+.. image:: images/tcm_start_instance_details.png
+    :width: 700
+    :align: center
+    :alt: Instance details in TCM
+
+To connect to the instance interactively and execute code on it, go to the **Terminal** tab.
+
+.. image:: images/tcm_start_instance_terminal.png
+    :width: 700
+    :align: center
+    :alt: Instance terminal in TCM
+
+Creating a space
+----------------
```

Comment:
In the `Managing the cluster in TCM`, we list the following use cases:

Checking the cluster state.
Creating a space.
Writing data.
Viewing data.

Doesn't this mean that the section structure should look as follows?

- Managing the cluster in TCM
   - Checking state
   - Connecting to the instance
   - Creating a space (now it is on the top level)
   - Writing data
   - Reading data
   - Checking replication

Reply:
Sure, thanks. This was an incorrect heading level.